packages/react/src/components/form/form.test.tsx

```diff
@@ -75,10 +75,11 @@ describe("#Form", () => {
   test("should show the success state on successful login", async () => {
     const logInMock = vi.fn(() => Promise.resolve(TEST_USER));
     const user = userEvent.setup();
+    const onSuccess = vi.fn();
```

Comment:
Maybe it would make sense to call the callback with the `user` instance that gets created on login success? It could make this more flexible.